### PR TITLE
Form-validations

### PR DESCRIPTION
**Pull request by sergiogval**
This feature add a JS code to check if the email address is lowercase written.

### CODE REVIEW

**Comment by sergiogval** [comment]
I had a mistake and submitted this PR to the Local Storage Milestone. Please reject the CR. I already submitted it in the proper CR for From Validations.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @sergiogval  👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



You have submitted a project, that <EXPLAIN WHY PROJECT IS INVALID>

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: formV.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+form.addEventListener('submit', (event) => {
+  if (!isLowerCase(email.value)) {
+    errorMsg.innerText = ERROR_MESSAGE;
+    email.classList.add('email-input-error');
+    event.preventDefault();
+  } else {
+    errorMsg.innerText = '';
+    email.classList.add('email-input-error');
+  }
+});
```
Comment (andersonlebon) [inline formV.js:19]: ## [OPTINAL]

- [ ]  To improve the design I would suggest you add some styling error interaction on the input that contains the validation error. Just consider it as a challenge that can improve your UI.  @sergiogval   [see the challenge here in its form](https://andersonlebon.github.io/Portfolio-w2/). 
image
image

**Review by andersonlebon — APPROVED** [review]
### Hi @sergiogval 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 
- Nice design :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**